src/components/RecipeBox.js

```diff
@@ -18,6 +18,15 @@ class RecipeBox extends Component {
     hoverHandler(){
       this.setState({isHovering: !this.state.isHovering})
     }
+    openMenuHandler(event){
+      event.preventDefault();
+
+      const name = event.target.id;
+      const menu = this.refs[name];
+      if(menu != null) {
```

Comment:
Zwłaszcza, że to nie byłby `null` tylko `undefined` gdyby `name` odwoływał się do czegoś czego nie ma.